doc/faq.rst

```diff
@@ -81,3 +81,69 @@ HV_RAM_SIZE is changed to 240M
     hex "Size of the RAM region used by the hypervisor"
     default 0x07800000 if PLATFORM_SBL
     default 0x0f000000 if PLATFORM_UEFI
+
+How to change the assigned output for one UOS instead of the default setting ?
+******************************************************************************
+
+BXT HW has three pipes and each pipe can have three or four planes which helps to
+display the overlay video. And it can support up to 3 monitors simultaneously.
+In order to assign the display monitors between SOS and UOS, some parameters are
+added to control the assign policy. It can help to simplify the assign policy
+while it provides the flexibility of configuring the pipe/planes for SOS/UOS under IOT scenario.
```

Comment:
I think it would be worth mentionning too that this is known as the "plane restriction" feature. There are other scenarii possible to overlay content from different VMs across all display outputs.